docs: Add Clerk integration guide (#6033)
supabase/supabase: web/docs/guides/integrations/clerk.mdx

---
id: clerk
title: 'Clerk'
description: 'This guide explains how to connect your Supabase database with Clerk, a powerful authentication provider built for the modern web.'
---

> This guide explains how to connect your Supabase database with [Clerk](https://clerk.dev), a powerful authentication provider built for the modern web.

Reviewer: ```suggestion
This guide explains how to connect your Supabase database with [Clerk](https://clerk.dev), an authentication provider built for the modern web.
```

Since these are Docs, it's probably better to remove any "marketing speak".

Clerk authenticates users, manages session tokens, and provides user management functionality that can be used in combination with the authorization logic available in Supabase through PostgreSQL Row Level Security (RLS) policies.

This guide assumes you have a Supabase account and database project already set up.

If you don't have a Clerk account, you can [create one now](https://dashboard.clerk.dev/sign-up).

## Step 1: Create JWT template

> The first step is to create a new Clerk application from your Clerk Dashboard if you haven't done so already. You can choose whichever authentication strategy and social login providers you prefer. For more information, check out our [Set up your application](https://docs.clerk.dev/popular-guides/setup-your-application) guide.

Reviewer: ```suggestion
The first step is to create a new Clerk application from your Clerk Dashboard if you haven't done so already. You can choose whichever authentication strategy and social login providers you prefer. For more information, check out Clerk's [guide](https://docs.clerk.dev/popular-guides/setup-your-application).
```

removing "our", since this is in the supabase docs
